### PR TITLE
Armstjc/fix mlb function names - Release Candidate 2

### PR DESCRIPTION
**Pull request by armstjc**

### CODE REVIEW

File: examples/test_mlbam.py [hunk 1]
```diff
@@ -285,14 +285,14 @@ def getCareerHittingStats(playerID=0,gameType="R"):
...
 
 import sportsdataverse as sdv
-df =sdv.mlb.getTransactionsInRange(startDate=20200901,endDate=20200914)
+df =sdv.mlb.load_mlbam_transactions(startDate=20200901,endDate=20200914)
 print(df)
 """
-print('getTransactionsInRange')
-print(sdv.mlb.getTransactionsInRange(startDate=20200901,endDate=20200914))
+print('load_mlbam_transactions')
+print(sdv.mlb.load_mlbam_transactions(startDate="09/01/2020",endDate="09/01/2020"))
```
Comment (akeaswaran) [inline examples/test_mlbam.py:292]: Providing two different date formats here

Comment (armstjc) [reply examples/test_mlbam.py:292]: That was a fix to reflect code changes that happened when this pull request was happening: https://github.com/sportsdataverse/sportsdataverse-py/pull/33

**Review by akeaswaran — APPROVED** [review]
:seemslegit:

**Comment by saiemgilani** [comment]
why do these have loads_? shouldn't it be like mlbam_transactions() ?

**Comment by armstjc** [comment]
> why do these have loads_? shouldn't it be like mlbam_transactions() ?

Because the function/API call requires a range of dates for the call to work, and there isn't another function like this in sdv-py.

**Comment by saiemgilani** [comment]
> > why do these have loads_? shouldn't it be like mlbam_transactions() ?
> 
> Because the function/API call requires a range of dates for the call to work, and there isn't another function like this in sdv-py.

Fine but I reserve the right to change the naming convention here if I feel like it is necessary

**Review by saiemgilani — COMMENTED** [review]
Fine but reserve the right to change

**Pull request merged**